packages/FormElement/stories/examples/AlternateLayouts.js

```diff
@@ -31,36 +42,64 @@ const ExampleStory = () => {
       </Heading>
       <Rule />
       <Tagline>Form Element with instructions component.</Tagline>
-      <Rule />
-      <FormElement label="Form Label">
+      <br />
+      <FormElement hasRequiredLabel={hasRequiredLabel} label="Form Label">
         <FormElement.Instructions>
           <span>
             Example text for extra panel for questionnaires. Example text for extra panel for questionnaires Example
             text for extra panel for questionnaires. Example text for extra panel for questionnaires
           </span>
         </FormElement.Instructions>
-        <Input onChange={handleChange} value={value} placeholder="Form placeholder" />
+        <FormElement.Content>
```

Comment:
Oof I can't say I have a better alternative in mind (except perhaps the `<FormElement.Input>` idea we discussed), but having to add this wrapper in every single form element seems too verbose.
I'm curious if others feel the same

Reply:
Having to use it as a render prop every time seems a bit cumbersome... but sometimes you might want the control.  What if we handle both? 

If `Content` has a function as `children`, do this, but if its `children` is just a single `node`, just clone the `idForLabel` + `ariaDescribedBy` onto it? 

And if it's multiple nodes... uh... just render them? Maybe it would be nice to have a way to also "just render them" even if it's a single node... but not sure what the API could look like for that 🤔

Reply:
...maybe `<FormElement.Content shouldAugmentProps={false}>`? 
It could be `true` by default?

Reply:
@mikrotron I think if you want to use the `idForLabel`, `ariaDescribedBy` or `labelRef` we should make the consumer make use of the `FormElement.Content` component to avoid confusion. If they do not need any of those props, as in the case they want to render multiple children, then we can just render them out without making use of `FormElement.Content`...wdyt?

Reply:
I feel the same, I don't think is a good idea to make the FormElement extend/clone their children components, the FormElement is a Component that will be used in multiple places and in many different scenarios, and doing all this (wrapper, cloning) seem odd to me and against the idea of composition. 

We are actually making the FormElement aware of all other input components instead of making it agnostic, which can be a red sign that we are doing something wrong. 

The only solid solution that I see is the FormElement use a renderProp function and pass down whatever we want to. Each component (input) is responsible to use whatever suite their case.

A renderProp work for 1 , 2 multiple scenario, yes is cumbersome but to me is the right pattern to use in this case. 

my two cents.

Reply:
Also, let's take into consideration that if we make this API incorrectly removing it from our ecosystem will be hard over time because it will be used multiple times per feature, is not like the sidepanel or modal which normally is 1 use per features.

Reply:
If so I'd seriously consider @tristanjasper's idea of producing thin wrappers like `<FormElement.Input>` that would hook into FormElement's context and remove the need for cloning or inspecting children.